R/layers-convolutional.R

```diff
@@ -552,8 +552,8 @@ layer_separable_conv_2d <- function(object, filters, kernel_size, strides = c(1L
 #' @family convolutional layers 
 #'   
 #' @export
-layer_separable_conv_1d <- function(object, filters, kernel_size, strides = c(1L, 1L), padding = "valid", data_format = NULL, 
-                                    depth_multiplier = 1L, activation = NULL, use_bias = TRUE, 
+layer_separable_conv_1d <- function(object, filters, kernel_size, strides = 1L, padding = "valid", data_format = NULL, 
```

Comment:
`strides` should use `1` rather than `1L` (realize that the original code had `c(1L, 1L)` but that was also incorrect).